implants/eldritch/src/pivot/port_scan_impl.rs

```diff
@@ -512,9 +483,11 @@ mod tests {
             handle_port_scan(test_cidr, test_ports.clone(), String::from("tcp"), 5)
         );
 
+        let mut listen_task_iter = listen_tasks.into_iter();
+        
         // Run both
         let (_a, _b, _c, actual_response) =
-            tokio::join!(listen_task1,listen_task2,listen_task3,send_task);
+            tokio::join!(listen_task_iter.next().unwrap(),listen_task_iter.next().unwrap(),listen_task_iter.next().unwrap(),send_task);
```

Comment:
`unwrap()` is a war crime 👀

Reply:
Gonna leave this since it's running in test code.